src/main/java/edu/cursor/springsecurity/controller/BookController.java

```diff
@@ -0,0 +1,29 @@
+package edu.cursor.springsecurity.controller;
+
+import edu.cursor.springsecurity.models.Book;
+import edu.cursor.springsecurity.service.IBookService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@AllArgsConstructor
+public class BookController {
+
+    private IBookService IBookService;
```

Comment:
A better approach to autowire, is using @RequiredArgsConstructor and initialize Beans in class as final.